pixels.go

```diff
@@ -0,0 +1,98 @@
+package waifu2x
+
+import (
+	"fmt"
+	"math"
+)
+
+type Pixels struct {
+	Width  int
+	Height int
+	Pix    []uint8 // 0以下を0, 255以上を255 とする
+}
+
+func NewPixels(width, height int) *Pixels {
```

Comment:
__[gometalinter]__
* [golint] exported function NewPixels should have comment or be unexported